### ISSUE TITLE
Option to reorder Scope before Type

### ISSUE THREAD

**Issue by DennisL68** [leonardoanalista/cz-customizable#100]
For some reason I always try to enter Scope first.
So I'd like an option to swap the order of the questions presented.

**Comment by leonardoanalista** [comment]
It feels like I implemented something like this already. Let me look...

**Comment by leonardoanalista** [comment]
no, not yet. I wrote something like "ask for breaking change first"

**Comment by DennisL68** [comment]
Ok, looking forward to this as new feature :)

**Comment by rewathkafley** [comment]
You can write your own adapter for this by referring [this link](https://github.com/commitizen/cz-conventional-changelog) as example.

**Comment by kjvelarde** [comment]
Maybe a schema / pattern based approach would be a good idea here. 
```
schema = "<type>: <body>"
schema_pattern = "(feature|bug fix):(\\s.*)"
```
e.g:
https://commitizen-tools.github.io/commitizen/customization/
